vars/cloudFoundryDeploy.groovy

```diff
@@ -251,14 +268,20 @@ def deployCfNative (config) {
             }
         }
 
-        def returnCode = sh returnStatus: true, script: """#!/bin/bash
+        def deployScript = """#!/bin/bash
             set +x
             set -e
             export HOME=${config.dockerWorkspace}
-            cf login -u \"${username}\" -p '${password}' -a ${config.cloudFoundry.apiEndpoint} -o \"${config.cloudFoundry.org}\" -s \"${config.cloudFoundry.space}\"
+            cf login -u \"${username}\" -p '${password}' -a ${config.cloudFoundry.apiEndpoint} -o \"${config.cloudFoundry.org}\" -s \"${config.cloudFoundry.space}\" ${config.mtaLoginParameters}
```

Comment:
My comment was more about deployOpts being a duplicate. 
I think mtaLoginParameters should be named loginParameters or similar as it is not specific to mta.

Reply:
name adjusted according to your proposal, also `apiParameters`.